File: codewars/8kyu/even-or-odd.py
# SOLUTION
# ==============================================================================
def even_or_odd(number):
    return "Even" if number % 2 == 0 else "Odd"

# TESTS
# ==============================================================================
if __name__ == "__main__":
    assert even_or_odd(1) == "Odd", "Test case 1 failed"
    assert even_or_odd(2) == "Even", "Test case 2 failed"
    assert even_or_odd(-1) == "Odd", "Test case 3 failed"
    assert even_or_odd(-2) == "Even", "Test case 4 failed"
    assert even_or_odd(0) == "Even", "Test case 5 failed"
    print("All test cases passed!")

File: codewars/8kyu/even-or-odd.cpp
#include <string>
#include <cassert>
#include <iostream>

std::string even_or_odd(int number) {
    return number % 2 == 0 ? "Even" : "Odd";
}

void runTests() {
    assert(even_or_odd(2) == "Even");
    assert(even_or_odd(3) == "Odd");
    assert(even_or_odd(0) == "Even");
    assert(even_or_odd(-1) == "Odd");
    assert(even_or_odd(-4) == "Even");
    std::cout << "All tests passed!" << std::endl;
}

int main() {
    runTests();
    return 0;
}
